sdk/cosmos/azure_data_cosmos/src/clients/database_client.rs

```diff
@@ -40,28 +41,63 @@ pub trait DatabaseClientMethods {
     async fn read(
         &self,
         options: Option<ReadDatabaseOptions>,
-    ) -> azure_core::Result<azure_core::Response<DatabaseProperties>>;
+    ) -> azure_core::Result<Response<DatabaseProperties>>;
 
     /// Gets a [`ContainerClient`] that can be used to access the collection with the specified name.
     ///
     /// # Arguments
     /// * `name` - The name of the container.
     fn container_client(&self, name: impl AsRef<str>) -> ContainerClient;
+
+    /// Returns the identifier of the Cosmos database.
+    fn id(&self) -> &str;
+
+    /// Executes a query against containers in the database.
+    ///
+    /// # Arguments
+    ///
+    /// * `query` - The query to execute.
+    /// * `options` - Optional parameters for the request.
+    ///
+    /// # Examples
+    ///
+    /// The `query` parameter accepts anything that can be transformed [`Into`] a [`Query`].
+    /// This allows simple queries without parameters to be expressed easily:
+    ///
+    /// ```rust,no_run
```

Comment:
Nit: no reason to state the `rust` block itself. It's always assumed, just ` ```no_run ` is fine...unless you're putting it into markdown and importing like I'm doing for READMEs.

Reply:
I didn't realize that, but it makes sense! I've put the `rust` tag in everywhere else, so if I do remove it, I'll just do a separate pass.